docs/index.rst

```diff
@@ -25,6 +25,7 @@ Understand messages, hold conversations, and connect to messaging channels and A
    user-guide/installation
    user-guide/rasa-tutorial
    user-guide/building-assistants
+   user-guide/prototype-an-assistant
```

Comment:
@abhilasharoy am I correct in saying this was supposed to replace the rasa tutorial?

Reply:
yes

Reply:
```suggestion
   user-guide/prototype-an-assistant
   user-guide/building-assistants
```